tutorial/python_agents/README.rst

```diff
@@ -0,0 +1,183 @@
+Python Agent Tutorial
+=====================
+This tutorial demonstrates how to implement a python agent in geopm.
+Specifically, the tutorial shows how to use an agent that can monitor
+application state and can write platform settings that are applied for the
+duration of an application run.
+
+This directory includes an example implementation of a python agent that
+optionally writes one or more GEOPM controls at the start of a run, and only
+monitors execution for the rest of the run. Take a look at the implementation
+in `static_agent.py <./static_agent.py>`_.
+
+Monitor-only execution
+----------------------
+First, let's try running an application without modifying any GEOPM controls.
+In this example, we are using the stress-ng tool to stress 20 CPUs for 5 seconds.
+We'll run stress-ng with ``--taskset 0-19`` so we know which CPUs to expect will
+run ``stress-ng``. This isn't strictly necessary, and is only being done to help
+us know where to expect most of our CPU load when walking through this
+tutorial.
+
+.. code-block:: sh
+
+    ./static_agent.py --geopm-report stress-monitor.report -- stress-ng --cpu 20 --taskset 0-19 --timeout 5
```

Comment:
Nit: you can wrap this as you did in the examples below (for consistency and to maintain the column limit)